<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kursovaya_rabota_OC</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Widget{
image: url(:/new/prefix1/1.png);
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>871</width>
     <height>391</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
	border: 3px solid;
	border-radius: 105px;
	border-color: rgb(53, 137, 193);
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>90</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(255,255,255);
border: 1px solid rgb(53, 137, 193);
border-radius: 20px;
}
QPushButton:hover{
background-color: rgb(194, 217, 232);
border: 1px solid rgb(53, 137, 193);
border-radius: 20px;
}
QPushButton:pressed{
background-color: rgb(194, 217, 232);
border: 3px solid rgb(53, 137, 193);
border-radius: 20px;
}</string>
    </property>
    <property name="text">
     <string>Расcчитать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border: 1px solid rgb(53, 137, 193);
border-radius: 20px;
}
QPushButton:hover{
background-color: rgb(194, 217, 232);
border: 1px solid rgb(53, 137, 193);
border-radius: 20px;
}
QPushButton:pressed{
background-color: rgb(194, 217, 232);
border: 3px solid rgb(53, 137, 193);
border-radius: 20px;
}</string>
    </property>
    <property name="text">
     <string>Открыть файл</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Начальное положение
страничных блоков</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	border: 1px solid #000000;
border-radius: 10px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Список обращений</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>82</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	border: 1px solid #000000;
border-radius: 10px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Количество
страничных блоков</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>170</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Искомая
страница</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Количество
прерываний</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>783</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
background-color: rgb(255, 255, 255);
border: 1px solid #000000;
border-radius: 10px;
}
</string>
    </property>
    <property name="placeholderText">
     <string>Пошаговый результат</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>871</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>135</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	border: 1px solid #000000;
border-radius: 10px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>16</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border: 1px solid #ffffff;
border-left-color: rgb(0,0,0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>479</x>
      <y>186</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*QSlider::handle:horizontal {
              border: 1px solid #000000;
                            border-radius: 6px;
                            background: rgb(38,190,255);
                        
                             }
                       QSlider::groove:horizontal {
                            background: rgb(63,72,204);
                            height: 6px;
                             }
                       QSlider::handle:horizontal:pressed {
                            background: rgb(232, 246, 255);
                             }*/
 QSlider::groove:horizontal {
     border: 0px solid rgb(53, 137, 193);
     height: 3px; /* по умолчанию бороздка расширяется до размеров ползунка. задав высоту она принимает фиксированный размер */
     background: rgb(53, 137, 193);
     margin: 0px 0;
 }

 QSlider::handle:horizontal {
     background: rgb(255, 255, 255);
/*image: url(:/new/prefix1/perem_clic_2.png);*/
     border: 1px solid rgb(53, 137, 193);
     width: 20px;
     margin: -10px 0; /* рукоятка располагается по умолчанию в прямоугольнике содержимого бороздки. Расширяется наружу от бороздки */
     border-radius: 11px;
 }
 QSlider::handle:horizontal:hover {
     background: rgb(194, 217, 232);
     border: 1px solid rgb(53, 137, 193);
     width: 20px;
     margin: -10px 0; /* рукоятка располагается по умолчанию в прямоугольнике содержимого бороздки. Расширяется наружу от бороздки */
     border-radius: 11px;
 }
 QSlider::handle:horizontal:pressed {
     background: rgb(194, 217, 232);
     border: 2px solid rgb(53, 137, 193);
     width: 20px;
     margin: -10px 0; /* рукоятка располагается по умолчанию в прямоугольнике содержимого бороздки. Расширяется наружу от бороздки */
     border-radius: 11px;
 }
/* QSlider::handle:horizontal:!enabled {
     background: rgb(240, 240, 240);
	
     border: 2px solid #000000;
     width: 20px;
     margin: -8px 0; рукоятка располагается по умолчанию в прямоугольнике содержимого бороздки. Расширяется наружу от бороздки 
     border-radius: 10px;
 }*/</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>185</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border: 1px solid rgb(53, 137, 193);
border-radius: 15px;
}
QPushButton:hover{
background-color: rgb(194, 217, 232);
border: 1px solid rgb(53, 137, 193);
border-radius: 15px;
}
QPushButton:pressed{
background-color: rgb(194, 217, 232);
border: 3px solid rgb(53, 137, 193);
border-radius: 15px;
}</string>
    </property>
    <property name="text">
     <string>Пауза</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>185</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border: 1px solid rgb(53, 137, 193);
border-radius: 15px;
}
QPushButton:hover{
background-color: rgb(194, 217, 232);
border: 1px solid rgb(53, 137, 193);
border-radius: 15px;
}
QPushButton:pressed{
background-color: rgb(194, 217, 232);
border: 3px solid rgb(53, 137, 193);
border-radius: 15px;
}</string>
    </property>
    <property name="text">
     <string>Сбросить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>291</x>
      <y>175</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Скорость воспроизведения</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>655</x>
      <y>190</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>50</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="Label_clic" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Label_clic" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>31</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Label_clic" name="label_4">
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>557</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Label_clic" name="label_5">
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>57</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Label_clic</class>
   <extends>QLabel</extends>
   <header>label_clic.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
